recipes/brb-seqtools/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.1" %}
+{% set name = "BRB-seqTools" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+build:
+  number: 0
+
+source:
+    fn: BRBseqTools.{{version}}.jar
+    url: https://github.com/DeplanckeLab/BRB-seqTools/releases/download/v.{{version}}/BRBseqTools.{{version}}.jar
+    sha256: 9557dc3793e2a21c93bdab7a8bc7cfde8b96f8251ffc7eec456ee125eb4ad331
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - openjdk >=8
+
+test:
+  commands:
+    - BRBseqTools
+
+about:
+  home: https://github.com/DeplanckeLab/BRB-seqTools
+  license: TBD
```

Comment:
Hold until  https://github.com/DeplanckeLab/BRB-seqTools/issues/2 is fixed

Reply:
The author of `BRB-seqTools` selected a Creative Commons [BY-NC-SA](https://github.com/DeplanckeLab/BRB-seqTools/blob/master/LICENSE.md) license. 
@bioconda/core: Does the non-commercial constraint prohibit inclusion in Bionconda? Creative Commons provides [this guidance](https://wiki.creativecommons.org/wiki/NonCommercial_interpretation) on the NonCommercial element.